pkg/wasp/shortage-detector/shortage_detector.go

```diff
@@ -60,7 +67,12 @@ func (sdi *ShortageDetectorImpl) ShouldEvict() (bool, error) {
 	averageSwapInPerSecond := float32(firstStat.SwapIn-secondNewest.SwapIn) / timeDiffSeconds
 	averageSwapOutPerSecond := float32(firstStat.SwapOut-secondNewest.SwapOut) / timeDiffSeconds
 	highTrafficCondition := averageSwapInPerSecond > sdi.maxAverageSwapInPagesPerSecond && averageSwapOutPerSecond > sdi.maxAverageSwapOutPagesPerSecond
-	overCommitmentRatioCondition := sdi.maxMemoryOverCommitmentBytes < firstStat.SwapUsedBytes-firstStat.AvailableMemoryBytes-firstStat.InactiveFileBytes
```

Comment:
Maybe we can remove SwapUsedBytes, AvailableMemoryBytes or InactiveFileBytes now or are they used in other places?

Reply:
You are correct! removed